file_link.rb

```diff
@@ -0,0 +1,15 @@
+
+module File_link
```

Comment:
module and class name should not have _
Also use appropriate name like `FileUtility` or `FileUtilities`